docs/user-guides/organization/creation.md

```diff
@@ -88,6 +88,8 @@ This ensures all pre-requisites are met, the organization is configured correctl
      mappedOrgAdminIdPGroup: Name of the group in the IDP that should be mapped to the organization admin role.
    ```
 
+The field `mappedOrgAdminIdPGroup` is mandatory.
```

Comment:
Instead of adding this addition here. Can you update line 98 to use `must be set to the `mappedOrgAdminIdPGroup` instead of `should be set to the `mappedOrgAdminIdPGroup`

Reply:
Done